### PR TITLE
Add require header for 4626

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: EIPS/eip-4626.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ status: Review
 type: Standards Track
 category: ERC
 created: 2021-12-22
+requires: 20, 165, 712
```
[the remark this answers is not in the document]
Comment (axic) [reply EIPS/eip-4626.md:11]: Even if it is mentioned as "SHOULD", the document depends on the 165/712/2612.

[the remark this answers is not in the document]
Comment (axic) [reply EIPS/eip-4626.md:11]: > We are discussing whether ERC165 should be required (leaning towards no)

Interesting, what is the reasoning for not having it?

[the remark this answers is not in the document]
Comment (fubuloubu) [reply EIPS/eip-4626.md:11]: > > We are discussing whether ERC165 should be required (leaning towards no)
> 
> Interesting, what is the reasoning for not having it?

It doesn't help very much except in one circumstance where it just helps avoid human error a little bit. Hence why we added as SHOULD.

**Review by fubuloubu — APPROVED** [review]
Approved per process

Will revisit necessity to have 712 or 165 in future PR

**Comment by transmissions11** [comment]
If anything we should specify 2612 instead of 712 no?

**Pull request merged**